src/tests/ftest/util/command_utils.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, namespace, command, path="", subprocess=False):
         self.verbose = True
         self.env = None
         self.sudo = False
+        self.sudo_E = False
```

Comment:
Have you considered defining `self.sudo_args` instead of `self.sudo`, where:

- `self.sudo_args = None` is equivalent to the current `self.sudo = False` behavior
- `self.sudo_args = ""` would add `"sudo"` to the command 
- `self.sudo_args = "-n -E"` would add `"sudo -n -E"` to the command 

If we wanted something that would have less impact to the current code:
```
    def __init__():
        ...
        self.sudo_args = None

    @property
    def sudo(self):
        return self.sudo_args is not None
        
    @sudo.setter
    def sudo(self, value):
        self.sudo_args = "-n" if value else None
```
Which would allow code to either do:

- `<ExecutableCommand>.sudo = True`             --> `sudo -n <command>`
- `<ExecutableCommand>.sudo_args = "-n"`     --> `sudo -n <command>`
- `<ExecutableCommand>.sudo_args = "-n -E"` --> `sudo -n -E <command>`

Reply:
`sudo_args` > `sudo_E`, but I'm going to take a rain check on that parameter, as it's no longer used in apricot/test.py.  3e2406951699031108a90fc15df68a95f661d349